src/backend/aspen/workflows/shared_utils/database.py

```diff
@@ -34,15 +36,23 @@ def create_temp_table(session: Session, source_table: Table) -> Table:
     return table_object
 
 
-def mv_table_contents(session: Session, source_table: Table, dest_table: Table) -> None:
+def mv_table_contents(
+    session: Session,
+    source_table: Table,
+    dest_table: Table,
+    filters: Optional[Iterable[ClauseElement]] = None,
+) -> None:
     """Deletes contents of dest, copies in contents from source to dest.
```

Comment:
Nit: I'd like to see the docstring updated to address that the use case is not always a drop all --> move all operation anymore.